workers/dotnet/App.cs

```diff
@@ -82,6 +82,13 @@ public static class App
         description: "Prometheus handler path",
         getDefaultValue: () => "/metrics");
 
+    /// <summary>
+    /// Main entry point.
+    /// </summary>
+    /// <param name="args">CLI args.</param>
+    /// <returns>Task for completion.</returns>
+    public static Task Main(string[] args) => RunAsync(args);
```

Comment:
The linter kept flagging that there's no `Main`. The tests pass with this so it seems good?